cmd/dagger/call.go

```diff
@@ -191,6 +182,17 @@ func writeOutputFile(path string, buf *bytes.Buffer) error {
 	return os.WriteFile(path, buf.Bytes(), 0o644) //nolint: gosec
 }
 
+// logOutputSuccess prints to stderr the output path to the user.
+func logOutputSuccess(cmd *cobra.Command, path string) {
+	path, err := filepath.Abs(path)
+	if err != nil {
+		// don't fail because at this point the output has been saved successfully
+		cmd.PrintErrf("WARNING: failed to get absolute path: %s\n", err)
+		path = outputPath
+	}
+	cmd.PrintErrf("Saved output to %q.\n", path)
```

Comment:
this was my bad since I didn't rebase with main before merging and this is something helder fixed recently. Will revert in a follow-up PR. Thx for bringing this up alex.